esp32_ulp/util.py

```diff
@@ -67,3 +68,12 @@ def validate_expression(param):
                 if c not in '0123456789abcdef':
                     state = 0
     return True
+
+
+def file_exists(filename):
+    try:
+        os.stat(filename)
+        return True
+    except:
+        pass
+    return False
```

Comment:
try/except/else? also: the exception raised - is it FileNotFoundError?

Reply:
Fixed. Catching OSError now.